CHANGELOG.md

```diff
@@ -64,6 +64,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
   - `polygon`
   - `nomics`
   - `openexchangerates`
+  - `coinmarketcap`
```

Comment:
Coinmarketcap is going to be merged throught this PR or through https://github.com/smartcontractkit/external-adapters-js/pull/218/ ?

Reply:
Other PR. This branch was rebased with that one.